pam/internal/adapter/nativemodel.go

```diff
@@ -800,19 +800,14 @@ func (m nativeModel) handleNewPassword() tea.Cmd {
 }
 
 func (m nativeModel) newPasswordChallenge(previousChallenge *string) tea.Cmd {
-	if previousChallenge == nil {
-		if cmd := maybeSendPamError(m.sendInfo("Insert '%[1]s' to cancel the request and go back",
-			nativeCancelKey)); cmd != nil {
-			return cmd
-		}
-	} else {
-		if cmd := maybeSendPamError(m.sendInfo("Repeat the previously inserted password or insert '%[1]s' to cancel the request and go back",
-			nativeCancelKey)); cmd != nil {
-			return cmd
-		}
+	challengeLabel := fmt.Sprintf("%[1]s (or use '%[2]s' to go back)",
```

Comment:
As a side note, the UX of "goind back" by typing "r" as the password is very weird. I expect that the instruction "type 'r'" makes it a little easier to understand than "use 'r'".

```suggestion
	challengeLabel := fmt.Sprintf("%[1]s (or type '%[2]s' to go back)",
```

Reply:
Yeah, I agree, I wanted (and I initially did it), then I decided to save an extra char :).

But well, it's not a big deal.